templates/components/streamfield/blocks/quote_block.html

```diff
@@ -1,5 +1,19 @@
-{% verbatim %}
-<div class="">
-    {{ self }}
-</div>
-{% endverbatim %}
+<div class="rich-text mb-10 md:mb-20 last:mb-0">
+    <h2 class="font-serif4 text-3xl md:text-5xl font-semibold text-center
+    {% if large_spacing %}
+        mb-10 md:mb-20
+    {% else %}
+        mb-8 md:mb-10
+    {% endif %}
+    ">
+        "{{ value.quote }}"</h2>
+    {% if value.attribution %}
+        <h2 class="font-serif4 text-base md:text-xl font-medium text-center
```

Comment:
If we use a heading, it probably makes more sense for this to be a lower-level heading i.e. h3 so it's under the quote's heading instead of being adjacent.

Reply:
I've switched everything so that we aren't using headings here anymore. See one of my prior comments.